selfdrive/athena/athenad.py

```diff
@@ -665,7 +665,11 @@ def stat_handler(end_event: threading.Event) -> None:
 def ws_proxy_recv(ws: WebSocket, local_sock: socket.socket, ssock: socket.socket, end_event: threading.Event, global_end_event: threading.Event) -> None:
   while not (end_event.is_set() or global_end_event.is_set()):
     try:
+      # TODO: this actually throws real exceptions because it recvs text sometimes
+      # kibana: `"bytes-like" AND ws_proxy_recv`
       data = ws.recv()
+      if isinstance(data, str):
+        data = data.encode("utf-8")
```

Comment:
@adeebshihadeh have any clue why this can sometimes be str? I would have ignored non-bytes and made a test